client/Lykke.Service.ConfirmationCodes.Client/Models/Request/SmsConfirmationRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Lykke.Service.ConfirmationCodes.Client.Models.Request
+{
+    /// <summary>
+    /// Basic request for sms confirmations
+    /// </summary>
+    public class SmsConfirmationRequest
+    {
+        /// <summary>
+        /// Phone number
+        /// </summary>
+        [Required]
+        [MinLength(8)]
```

Comment:
Is 8 really enough?

Reply:
https://stackoverflow.com/questions/14894899/what-is-the-minimum-length-of-a-valid-international-phone-number